_episodes/11-connecting.md

```diff
@@ -327,7 +333,7 @@ supplied by the instructors.
 ```
 {: .language-bash}
 
-You may be asked for your password. Watch out: the characters you type after
+You may be asked for your password, or a One Time Passcode for Multi Factor Authentication. Watch out: the characters you type after
```

Comment:
```suggestion
You may be asked for your password, and/or a One Time Passcode for Multi Factor Authentication. Watch out: the characters you type after
```

I think if public-key authentication fails you are prompted for a password then the OTP.